cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java

```diff
@@ -149,15 +150,7 @@
     public static final String DEFAULT_COMMITLOG_SYNC = "periodic";
     public static final int DEFAULT_COMMITLOG_SYNC_PERIOD_IN_MS = 10000;
     public static final int DEFAULT_COMMITLOG_SEGMENT_SIZE_IN_MB = 32;
-    public static final List<Map<String, Object>> DEFAULT_SEED_PROVIDER =
-            ImmutableList.<Map<String, Object>>of(
-                    ImmutableMap.<String, Object>of(
-                            "class_name",
-                            "org.apache.cassandra.locator.SimpleSeedProvider",
-                            "parameters", ImmutableList.of(ImmutableMap.of
-                                    ("seeds", "127.0.0.1"))
-                    )
-            );
+    public static final String DEFAULT_SEEDS_URL = "http://cassandra.marathon.mesos:8080/v1/seeds";
```

Comment:
Okay I'm merging this to master. I've already pulled this into the 3.0.7 branch and its testing there. I'll rebase off of master.